deployment/chart/Chart.yaml

```diff
@@ -15,15 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.2.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "3.0.0"
-
-dependencies:
-  - name: postgresql 
```

Comment:
Do we no longer have a dependency on PostgreSQL or is it expressed in another way?

Also, let's use Posgresql version 16 if possible.

Reply:
The database is no longer packaged with the app, but rather is a CloudSQL instance that is a bit more resilient. Even if I wanted to continue having the db in k8s, I probably wouldn't tie the deployments together like this.

I can update the terraform PR to use 16.